src/app/components/App.tsx

```diff
@@ -85,9 +86,11 @@ function PlannerList() {
             }}
             key={id}
           >
-            <PlannerProvider plannerId={id} title={title} order={index}>
-              <Planner isActive={activePlanner === id} />
-            </PlannerProvider>
+            <MajorVerificationProvider plannerId={id}>
```

Comment:
Why do we need a MajorVerificationProvider for each planner?

Reply:
Because Major Verification depends on each planner, having only one MajorVerificationProvider would require passing in all of the user's planners which would result in awkward code.